examples/ingest/slack/ingest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Processes the Unstructured-IO/unstructured repository
+# through Unstructured's library in 2 processes.
+
+# Structured outputs are stored in slack-ingest-output/
+
+# oldest, latest arguments are optional
+
+# Ingests a slack text channel into a file.
```

Comment:
Maybe add a hint that the token associated with the app must be in a channel to be able to ingest from it. And, it is channel id, not channel name.